### PR TITLE
Add CloudFlareService.delete_custom_hostname_by_name().

### PR DESCRIPTION
**Pull request by github-killough-org**
yo

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
We need to resolve conflicts first.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: pycloudflare/services.py [hunk 1]
```diff
@@ -190,12 +190,16 @@ def get_custom_hostname_by_name(self, zone_id, hostname):
             raise CustomHostnameNotFound()
 
         return result[0]
-
+
```
Comment (snitch) [inline pycloudflare/services.py]: W293 blank line contains whitespace

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by kosmos342 — APPROVED** [review]

**Pull request merged**